src/llvm-cpufeatures.cpp

```diff
@@ -57,7 +58,7 @@ bool have_fma(Function &intr, Function &caller) {
 
     Attribute FSAttr = caller.getFnAttribute("target-features");
     StringRef FS =
-        FSAttr.isValid() ? FSAttr.getValueAsString() : jl_TargetMachine->getTargetFeatureString();
+        FSAttr.isValid() ? FSAttr.getValueAsString() : jl_ExecutionEngine->getTargetMachine().getTargetFeatureString();
```

Comment:
We should probably get the `TLI` (e.g. TargetLoweringInfo) from the module and use that instead. Otherwise this pass can not be executed from `opt`.

Reply:
Let's keep this for now (in this PR), and fix that later.